src/tools/illink/test/Mono.Linker.Tests.Cases/Symbols/Dependencies/LibraryWithMdb/compile.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Run this script from a VS command prompt
+
+BASEDIR=$(dirname $0)
+mcs /debug:full /out:$BASEDIR/LibraryWithMdb.dll /target:library $BASEDIR/LibraryWithMdb.cs
```

Comment:
Dll's should not be checked into the repository.  You might want to move this to runtime-assets.